Dangerfile.swift

```diff
@@ -5,14 +5,11 @@ let danger = Danger()
 let pr = danger.github.pullRequest
 let isReleasePr = pr.head.ref.hasPrefix("release")
 
-func isConventionalCommitTitle() -> Bool {
-    // Commitizen-compatible conventional commit titles
-    pr.title.hasPrefix("BREAKING CHANGE:") ||
-    pr.title.hasPrefix("chore:") ||
-    pr.title.hasPrefix("fix:") ||
-    pr.title.hasPrefix("feat:")
-}
+// MARK: - Conventional Commit Title
```

Comment:
@jnewc This update allows more prefixes, like `ci:` - would you mind updating 3DS and Nol, and get everything aligned?